website/content/docs/commands/pki/issue.mdx

```diff
@@ -0,0 +1,71 @@
+---
+layout: docs
+page_title: pki issue - Command
+description: |-
+  The "pki issue" command issues a new intermediate ("issuer") certificate,
+  using a parent certificate in vault.
+---
+
+# pki issue
+
+This command creates a intermediate certificate authority certificate signed by
+the `<parent>` in the `<child_mount>`, using the options to determine the fields
+on that certificate.
+
+## Usage
+
+Usage: `vault pki issue [flags] <parent> <child_mount> [options]`
+
+- [flags] are optional arguments described below
```

Comment:
```suggestion
- `[flags]` are optional arguments described below
```

Just a suggestion for consistency. I'd apply it to the rest as well if you apply it here. :-)